metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -167,6 +166,34 @@ export function getNodeAutoInstrumentations(
   return instrumentations;
 }
 
+function checkManuallyProvidedInstrumentationNames(
+  manuallyProvidedInstrumentationNames: string[]
+) {
+  for (const name of manuallyProvidedInstrumentationNames) {
+    if (!Object.prototype.hasOwnProperty.call(InstrumentationMap, name)) {
+      diag.error(`Provided instrumentation name "${name}" not found`);
+    }
+  }
+}
+
+/**
+ * Returns the list of instrumentations that are allowed based on the environment variable.
```

Comment:
```suggestion
 * Returns the list of instrumentations that are enabled based on the environment variable.
```

Reply:
done with [this commit](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1953/commits/5f7a4cf6698b210bbc59e65fcda6e4c05e73b804), that is squashed into [the feature commit](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1953/commits/c7393d9ec7dddbdc7031d37a420d69fd30e11614) after rebasing on top of latest main